lib/travis/build/addons/srcclr.rb

```diff
@@ -0,0 +1,29 @@
+require 'shellwords'
+
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class Srcclr < Base
+        SUPER_USER_SAFE = true
+
+        def before_finish
+          srcclr_config = @config.to_s.strip.shellescape
```

Comment:
The expected way to use the addon in `.travis.yml` is `srcclr: 'true'`. `#to_s` was used in case of a usage error where user did not pass in any values i.e. `srcclr: `. In such a case, the `config` is of `Hash` type instead of a `String` type, and calling `#strip` will throw an error.

I have removed `#to_s` as the user is expected to pass in the correct values now.